packages/@uppy/companion/src/uppy.js

```diff
@@ -218,7 +233,7 @@ const getOptionsMiddleware = (options) => {
       options,
       s3Client,
       authToken: req.header('uppy-auth-token') || req.query.uppyAuthToken,
-      clientVersion: req.header('uppy-versions') || req.query.uppyVersions,
+      clientVersion: req.header('uppy-versions') || decodeURIComponent(req.query.uppyVersions) || '1.0.0',
```

Comment:
This still resolves to `undefined` instead of `1.0.0` in my tests, because, I think:

`typeof req.header('uppy-versions')` returns `undefined`
`typeof decodeURIComponent(req.query.uppyVersions)` returns `string` 😱
`console.log(decodeURIComponent(req.query.uppyVersions))` returns `undefined`

Reply:
good spot, `decodeURIComponent(undefined) === 'undefined'`